src/utils/decorators/input/input.scss

```diff
@@ -13,7 +13,7 @@
     margin-top: 10px;
   }
 
-  &:hover .common-input__field .common-input__input {
+  .common-input__field .common-input__input:hover {
```

Comment:
I'm afraid I don't know the history of why such a specific rule existed, and because of that wasn't confident of reducing the specificity without knowing all the possible side effects. `common-input__input` is used in quite a few places.

Reply:
Actually there's an `input-outline` mixin applied for `:focus` and `:active`: https://github.com/Sage/carbon/blob/master/src/utils/decorators/input/input.scss#L93

Maybe we should just extend that rule and apply the same mixin on `:hover` too?

Reply:
Yeah, this looks like a much better solution. I'm just pushing the change as it looks OK clicking around.